edgesite/pkg/controller/config/kube.go

```diff
@@ -0,0 +1,101 @@
+package config
+
+import (
+	"time"
+
+	"github.com/kubeedge/beehive/pkg/common/config"
+	"github.com/kubeedge/beehive/pkg/common/log"
+	"github.com/kubeedge/kubeedge/edgesite/pkg/controller/constants"
+)
+
+// KubeMaster is the url of edge master(kube api server)
+var KubeMaster string
```

Comment:
`KubeAPIEndpoint`?

Reply:
This is the same as existing cloud side controller. I am keeping as is.